src/interpreter/Node.h

```diff
@@ -401,6 +389,25 @@ class ConditionalOperation {
     Own<Node> cond;
 };
 
+/**
+ * @class RelationalOperation
+ * @brief Interpreter operation that holds a single relation
+ */
+class RelationalOperation {
+public:
+    using RelationHandle = Own<RelationWrapper>;
```

Comment:
If you want the typedef, perhaps 

```suggestion
    using RelationHandle = RelationWrapper*;
```

instead?